src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -72,6 +85,19 @@ class OutcomeCardContainer extends Component {
     });
   };
 
+  isEditing = () => {
+    return !this.state.activeEditingClients.isEmpty()
+  }
+
+  trackEditingClient = (uid, isEditing) => {
+    this.setState((curState) => {
```

Comment:
Coming from Elm/Redux where data updates are always functional, this took me an embarrassing amount of time to debug :P